distributed/tests/test_spill.py

```diff
@@ -11,72 +12,132 @@ def test_spillbuffer(tmpdir):
     assert buf.memory is buf.fast
     assert buf.disk is buf.slow
 
-    assert not buf.spilled_by_key
-    assert buf.spilled_total == 0
+    assert not buf.slow.weight_by_key
+    assert buf.slow.total_weight == 0
 
     a, b, c, d = "a" * 100, "b" * 100, "c" * 100, "d" * 100
-    s = sizeof(a)
+    s = sizeof(a)  # this is size of a in memory
+
+    pickle_size = sum(len(frame) for frame in serialize_bytelist(a))
+
     # Test assumption made by this test, mostly for non CPython implementations
     assert 100 < s < 200
 
     buf["a"] = a
     assert not buf.disk
-    assert not buf.spilled_by_key
-    assert buf.spilled_total == 0
+    assert not buf.slow.weight_by_key
+    assert buf.slow.total_weight == 0
```

Comment:
```suggestion
    assert buf.fast.weights == {"a": inmem_size}
    assert buf.fast.total_weight == inmem_size
    assert buf.slow.weight_by_key == {}
    assert buf.slow.total_weight == 0
```

1. Could you please replace everywhere weight_by_key with weight to align Slow with the same attribute on LRU?
2. Please update all checks below to match this one